src/core/lib/iomgr/tcp_server_uv.c

```diff
@@ -76,13 +76,30 @@ struct grpc_tcp_server {
 
   /* shutdown callback */
   grpc_closure *shutdown_complete;
+
```

Comment:
I don't know what the docstring should say. I just copied this from the windows implementation, and the other implementations don't say.